docs/infra/vulnerability-management.md

```diff
@@ -0,0 +1,31 @@
+# Vulnerability Management for Docker Images
+This repository contains a GitHub action that allows you to scan Docker images for vulnerabilities. The action, named `image-scan` is located in the directory `.github/actions/image-scan`. The goal in scanning the image before pushing it to the repository is so that you can catch any vulnerabilities before deploying the image, ECR scanning takes time and the image can still be used even with vulnerabilities found by Inspector. Add this to be part of the PR process so that any pushes to the repo will trigger the scan
```

Comment:
I'm wondering, when should these scans be run? I'm guessing they should be run on PRs. Should they also run on merges to main, or is that redundant?

Reply:
I'm wondering the same thing right now, I think PRs for sure - dependabot is trying to merge a PR that has a vulnerability for us right now. But, I think it should on a merge to main **IF** the last scan of the image is like X days before the merge to main. That way, in case there was something that was found between the last push to the PR and the merge, it should get caught

Reply:
That makes sense. In that case I think `on: pull_request` and `on: push: [main]` is probably good. No harm in running on every merge